File: DotNet/Node.Core/Biz/Interfaces/GetStatus/IProcess.cs
using System;

using Node.Core.Biz.Manageable.Parameters;

namespace Node.Core.Biz.Interfaces.GetStatus
{
    /// <summary>
    /// The Interface for GetStatus Process
    /// </summary>
    public interface IProcess
    {
        /// <summary>
        /// Process for GetStatus.
        /// </summary>
        /// <param name="token">The Token which is result of Authentication Operation.</param>
        /// <param name="transID">The transaction id is result of Solicict/Submiet Operation.</param>
        /// <param name="param">Operation related parameters</param>
        /// <returns>Status String</returns>
        string Execute(string token, string transID, ProcParam param);
    }
}

File: EN Node for Java environment/Node/src/Node/Biz/Interfaces/Notify/IProcess.java
package Node.Biz.Interfaces.Notify;

import java.rmi.RemoteException;

import Node.Biz.Custom.ProcParam;
import Node.WebServices.Document.ClsNodeDocument;
/**
 * <p>This class create Notify Process Interface.</p>
 * <p>Company: enfoTech & Consulting, Inc.</p>
 * @author enfoTech
 * @version 2.0
 */

public interface IProcess {
	/**
	 * Execute
	 * @param token The authentication token
	 * @param nodeAddress The node address of notification
     * @param dataFlow The data flow name
     * @param docs The notification file object array 
	 * @param param The process parameter 
	 * @throws RemoteException
	 * @return String The status
	 */
  public String Execute (String token, String nodeAddress, String dataFlow, ClsNodeDocument[] docs, ProcParam param) throws RemoteException;
}
